src/libraries/Common/src/System/Security/Cryptography/CompositeMLDsaManaged.cs

```diff
@@ -709,15 +709,15 @@ private static Dictionary<CompositeMLDsaAlgorithm, AlgorithmMetadata> CreateAlgo
                     CompositeMLDsaAlgorithm.MLDsa87WithRSA3072Pss,
                     new AlgorithmMetadata(
                         MLDsaAlgorithm.MLDsa87,
-                        new RsaAlgorithm(3072, HashAlgorithmName.SHA512, RSASignaturePadding.Pss),
+                        new RsaAlgorithm(3072, HashAlgorithmName.SHA256, RSASignaturePadding.Pss),
```

Comment:
id-MLDSA87-RSA3072-PSS-SHA512 says "RSASSA-PSS with SHA384".

But, I assume that since you changed this, it means that a test actually shows it using RSASSA-PSS with SHA256?

Sounds like they need a spec-or-kat bug.